### PR TITLE
Use ES6 standards

### PR DESCRIPTION
**Pull request by juampynr**
I wanted to learn how to install and use eslint with the airbnb standard so I created my own implementation. I managed to set up an npm script and set the standard in PhpStorm.

### CODE REVIEW

**Comment by juampynr** [comment]
Based on https://github.com/juampynr/elasticsearch-nodejs/pull/1

File: app/.eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "extends": "airbnb-base",
```
Comment (justafish) [inline app/.eslintrc.json:2]: I think this should be just "extends": "airbnb"

Comment (juampynr) [reply app/.eslintrc.json:2]: I looked at the difference between https://www.npmjs.com/package/eslint-config-airbnb and https://www.npmjs.com/package/eslint-config-airbnb-base and found out that `airbnb` contains useful tools for React projects, while `airbnb-base` is recommended for projects that don't use React. See this comment:
image

Based on that, is then what I have correct @justafish?

Comment (justafish) [reply app/.eslintrc.json:2]: oh cool, I didn't know that! Thanks @juampynr ! 💃

**Pull request merged**